bin/inx-chronicle/config.template.toml

```diff
@@ -7,6 +7,8 @@ database_name = "chronicle"
 [api]
 port = 8042
 allow_origins = "0.0.0.0"
+password_hash = "c42cf2be3a442a29d8cd827a27099b0c" # "password"
```

Comment:
We should also explicitly add an expiration field for the JWT here. By default the expiration should be set to never expire, with a comment to explain that this should be changed in production.

Reply:
Still not sure about the rationale of defaulting to never expire, but I'll add the field.